liferay-echo/liferay-portal
# LPS-185805 Migrate utils to TypeScript

Motivation
=======
This is part of a task whose objective is to migrate the js code of analytics-reports-web (Content Performance Panel) to Typescript.
[Link to story or ticket](https://issues.liferay.com/browse/LPS-185805)

Steps to Verify:
----------------
1. To fake some data in th Content Performance Panel cherry-pick this commit or copy the code https://github.com/beltranrengifo/liferay-portal/pull/3/commits/c6023dbb41de97eb176a2201747451d4e5dcca0d
1. Open the Content Performance Panel and check that everything works ok
